lib/ts/Arena.h

```diff
@@ -166,4 +167,23 @@ Arena::str_store(const char *str, size_t len)
   return mem;
 }
 
+struct Arena_NoCache : public Arena {
+  std::allocator<uint64_t> m_alloc;
+
+  inkcoreapi void *
+  alloc(size_t size, size_t alignment = sizeof(double))
+  {
+    return m_alloc.allocate((size + 1) / sizeof(uint64_t));
+  }
+  void
+  free(void *mem, size_t size)
+  {
+    m_alloc.deallocate(static_cast<uint64_t *>(mem), size);
+  }
+};
+
+#if HAVE_LIBJEMALLOC
+#define Arena Arena_NoCache
```

Comment:
.. nope.   C++ blocks all signature overloads from a base class that have the same function name.

Lots and lots and lots of different techniques of specialized memory-hoarding doesn't speed things up, but often makes them vulnerable to unchecked and invisible bad-pointer usage.

There's also no way to profile deletions or allocations ... when they're hidden deep inside a memory-hoarder.   The thread-cache in Jemalloc is far more efficient.